features/step_definitions/provider_steps.rb

```diff
@@ -237,6 +237,8 @@ def setup_provider(login)
   set_current_domain(@provider.external_admin_domain)
   stub_integration_errors_dashboard
 
+  @provider.users.first.user_sessions.create! # Prevents welcome flash from showing up
```

Comment:
I understand the reason, but doesn't this make the `login` param useless? it will create a session anyway... and doesn't this break any test?